charts/argo-rollouts/templates/controller/clusterrole.yaml

```diff
@@ -266,6 +266,7 @@ rules:
     - list
     - watch
     - update
+    - patch
```

Comment:
Since this block is for contour, I wonder we can provide other block for trafficRouterPlugins? 🤔 

```yaml
{{- if .Values.controller.trafficRouterPlugins }}
- apiGroups:
    - projectcontour.io
  resources:
    - httpproxies
  verbs:
    - patch
{{- end }}
```